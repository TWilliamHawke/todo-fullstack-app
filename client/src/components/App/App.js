import React from 'react'
import { Provider } from 'react-redux'

import Routes from 'Src/routes';
import { store } from 'Src/redux/store';
import ErrorBoundary from '../ErrorBoundary/ErrorBoundary';


const App = () => {
  return (
    <Provider store={store}>
      <ErrorBoundary>
      <div className="todo-app">
        <Routes />
      </div>
      </ErrorBoundary>
    </Provider>
  )
}

export default App
