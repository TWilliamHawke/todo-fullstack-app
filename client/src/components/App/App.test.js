import React from 'react'
import ReactDOM from 'react-dom'
import App from './App'

import { shallow, render, mount } from 'enzyme'
import tojson from 'enzyme-to-json'


it('renders correctly', () => {
  const wrapper = shallow(<App />)

  expect(tojson(wrapper)).toMatchSnapshot()
})
