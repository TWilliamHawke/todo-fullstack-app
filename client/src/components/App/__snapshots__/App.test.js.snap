// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <ErrorBoundary>
    <div
      className="todo-app"
    >
      <Connect(Routes) />
    </div>
  </ErrorBoundary>
</Provider>
`;
